e2e/specs/nuxtPrint.cy.js

```diff
@@ -6,34 +6,35 @@ describe('Nuxt print test', () => {
   it('shows print preview', () => {
     cy.login('test@example.com')
 
-    cy.request(Cypress.env('API_ROOT_URL') + '/camps.jsonhal').then((response) => {
-      const body = response.body
-      const campUri = body._links.items[1].href
-      const camp = body._embedded.items[1]
-
-      const printConfig = {
-        language: 'en',
-        documentName: 'camp',
-        camp: campUri,
-        contents: [
-          {
-            type: 'Cover',
-            options: {},
-          },
-        ],
+    cy.request(Cypress.env('API_ROOT_URL') + '/camps/3c79b99ab424.jsonhal').then(
```

Comment:
why did you change this?

Reply:
Couldn't remember why. Reverted with commit https://github.com/ecamp/ecamp3/pull/3610/commits/a05fdec3237169dcf8c7003a77bc64921b9ca643